vnext/Shared/AsyncStorage/StorageFileIO.cpp

```diff
@@ -140,6 +140,10 @@ void StorageFileIO::append(const std::string &fileContent) {
   fwrite(fileContent.c_str(), sizeof(char), fileContent.size(), m_storageFile.get());
 }
```

Comment:
Consider doing the fflush here for safety's sake.  Later we should strongly consider reimplementing this entire module.

Reply:
Hal is adding a sqlite backed module with his Win32 Playground change that should also work for UWP. It sounded like there was some interest in this module becoming the basis for the community module instead of our current?

Reply:
We should try to leave implementation consistent once we have a community module though. Unless we add tooling for migration, any change will require apps to drop user data when upgrading.

Reply:
Yes, I am going to add a tracking issue for migrating to SqlLite after move to community module with current implementation (with this fix).

---
In reply to: [381317708](https://github.com/microsoft/react-native-windows/pull/4114#discussion_r381317708) [](ancestors = 381317708)

Reply:
I think the data drop issue will exist regardless whether the code is in RNW or community module, it will break once we migrate to sqlLite unless we add logic to read the storage file and save to sqlLite. I think that logic should be pretty simple to implement.

---
In reply to: [381319360](https://github.com/microsoft/react-native-windows/pull/4114#discussion_r381319360) [](ancestors = 381319360)

Reply:
I added a flush function separately in this module, so in case caller can do multiple append() followed by one flush(). 

---
In reply to: [380996960](https://github.com/microsoft/react-native-windows/pull/4114#discussion_r380996960) [](ancestors = 380996960)